news.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="New and Updates of Gandhi Associates">
+    <meta name="description" content="Get latest news and updates here.Gandhi Associates buying shares of : KIOCL Ltd,Bharti Telecom Ltd and  many more...">
```

Comment:
Update content 

Get the latest news and updates here. Gandhi Associates buying shares of KIOCL Ltd, Bharti Telecom Ltd, and many more...